clients/runtime/src/tests.rs

```diff
@@ -79,34 +81,6 @@ async fn test_too_low_priority_matching() {
 	assert!(err.is_pool_too_low_priority().is_some())
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn test_subxt_processing_events_after_dispatch_error() {
-	let (client, _tmp_dir) = default_provider_client(AccountKeyring::Alice).await;
-	let parachain_rpc = setup_provider(client.clone(), AccountKeyring::Alice).await;
-
-	let oracle_provider = setup_provider(client.clone(), AccountKeyring::Bob).await;
-	let invalid_oracle = setup_provider(client, AccountKeyring::Dave).await;
-
-	let event_listener = crate::integration::assert_event::<FeedValuesEvent, _>(
-		Duration::from_secs(80),
-		parachain_rpc.clone(),
-		|_| true,
-	);
-
-	let key = OracleKey::ExchangeRate(DEFAULT_TESTING_CURRENCY);
-	let exchange_rate = FixedU128::saturating_from_rational(1u128, 100u128);
-
-	let result = tokio::join!(
-		event_listener,
-		invalid_oracle.feed_values(vec![(key.clone(), exchange_rate)]),
-		oracle_provider.feed_values(vec![(key, exchange_rate)])
-	);
-
-	// ensure first set_exchange_rate failed and second succeeded.
-	result.1.unwrap_err();
-	result.2.unwrap();
-}
-
```

Comment:
@cr4pt0 what about this?

Reply:
@ebma 
Because there is no need in this tests because there is no Event about feeding in oracle spacewalk.
It is not relevant already. `FeedValuesEvent` does not exist in new redesigned oracle in spacewalk.

Reply:
Yes that might be true. But that was not the point of my comment.
I was trying to say that although the actual implementation of this test is using the Oracle, this might just be coincidental and they could have used any other extrinsics to test it. The name of the test 'test_subxt_processing_events_after_dispatch_error' indicates exactly this; that it is not about the oracle but about testing something about subxt. Hence why I was asking if we can change the implementation and keep the test.

Reply:
I think the main reason they used the oracle pallet for this test is that it is easy to execute one extrinsic successfully and let another fail. And you said that we don't have the `FeedValuesEvent` but that is not 100% true since we do have this event but it is only emitted in tests. Since this is a test however, it should be emitted, no?

Reply:
No. `FeedValuesEvent` emitted only for Mock runtime. for tests.
In `test_subxt_processing_events_after_dispatch_error` function feed_values calls DIA oracle extrinsic and not an spacewalk oracle. 
Seriously. other tests did the same. 
For example `test_automatic_issue_execution_succeeds` did 
```
assert_event::<ExecuteIssueEvent, _>(TIMEOUT, user_provider.clone(), move |x| {
	x.vault_id == vault_id.clone()
})
.await;
```
and create the user_provider
`let user_provider = setup_provider(client.clone(), AccountKeyring::Dave).await;`
I do not see any reason to keep this function or reimplement it. Because the original purpose of 
`test_subxt_processing_events_after_dispatch_error` was to check that the is an error when feed oracle with incorrect oracle provider. 
@ebma

Reply:
To be sure what the original purpose of the function was we would have to ask developers from interlay. I still argue that the name of the test, that is 'test_subxt_processing_events_after_dispatch_error' tells us its purpose and not its implementation. 
But if you don't want to re-implement it, then that's okay and we don't. I just want to make sure you get my point.